pkg/kritis/util/whitelist.go

```diff
@@ -49,8 +48,9 @@ func imageInWhitelist(image string) (bool, error) {
 		if err != nil {
 			return false, err
 		}
+
 		// Make sure images have the same context
```

Comment:
It's better to update this comment: `Make sure images have the same name`
"context" includes image tag & digest 👓

Reply:
Updated the comment in c858860